test/integration/EmptySetProposal003.test.ts

```diff
@@ -0,0 +1,131 @@
+import HRE from 'hardhat'
+import { expect } from 'chai'
+import { Signer } from 'ethers'
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers'
+
+import {
+  GovernorAlpha2,
+  EmptySetGovernor__factory,
+  EmptySetGovernor,
+  GovernorAlpha2__factory,
+  EmptySetTimelock__factory,
+  EmptySetTimelock,
+  EmptySetRegistry,
+  EmptySetRegistry__factory,
+} from '../../types/generated'
+import { govern, impersonate, time } from '../testutil'
+import { EMPTYSET_CONTRACTS } from '../../proposals/emptyset/contracts'
+import { ES_003 } from '../../proposals/emptyset/es003'
+
+const { ethers, deployments } = HRE
+const PROPOSER_ADDRESS = '0x589CDCf60aea6B961720214e80b713eB66B89A4d' // Equilibria Multisig
+
+const USE_REAL_DEPLOY = false
+const FORK_BLOCK = 14994972
+
+describe.only('Empty Set Proposal 003', () => {
```

Comment:
nit: remove `only`

Reply:
Also, can we pull the constant values that were changed to ensure they are set to what we expect?

Reply:
nit: still have an `only` here 😅